solarized-definitions.el

```diff
@@ -195,6 +195,11 @@ contains Solarized symbols."
                              (cond (solarized-degrade     3)
                                    (solarized-broken-srgb 2)
                                    (t                     1)))
+	   ,@(dark-and-light '((min-colors 16777216))
```

Comment:
I wonder if `(min-colors 16777216)` can just replace `(type graphic)` in the previous expression.

My non-terminal Emacs claims it has 16777216 colors (as I would expect), so this might just be a better characteristic to check for.

Reply:
Yes, that works just as well. I've updated the PR to just use that test for truecolor.